transport/grpc/inbound.go

```diff
@@ -80,6 +80,21 @@ func (i *Inbound) SetRouter(router transport.Router) {
 	i.router = router
 }
 
+// Addr returns the address on which the server is listening.
+//
+// Returns nil if Start has not been called yet
+func (i *Inbound) Addr() net.Addr {
+	i.lock.RLock()
+	defer i.lock.RUnlock()
+	// i.server is set in start, so checking against nil checks
+	// if Start has been called
+	// we check if i.listener is nil just for safety
+	if i.server == nil || i.listener == nil {
```

Comment:
If you're checking if `i.listener == nil` here, you should also add such checks in `newInbound()` and `start()`

Reply:
Sure, but this is just a small PR adding this functionality. I might go back in and add.

Reply:
Ok, LGTM then.